staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -633,12 +638,11 @@ func getNewItemFunc(listObj runtime.Object, v reflect.Value) func() runtime.Obje
 	}
 }
 
-func (s *store) Stats(ctx context.Context) (stats storage.Stats, err error) {
-	if s.stats != nil {
-		stats, err := s.stats.Stats(ctx)
-		if !errors.Is(err, errStatsDisabled) {
-			return stats, err
-		}
+func (s *store) Stats(ctx context.Context) (storage.Stats, error) {
+	stats := s.getStatsCollector()
```

Comment:
can we name the variable and getter to make it clearer this is specific to resource size stats ... and that without that getter, we still return stats, they just don't include resource size data

```go
if collector := s.getResourceSizeStatsCollector(); collector != nil {
  return collector.Stats(ctx)
}
// existing code path to just return count stats that don't include resource size
...
```

Reply:
Done